connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -404,6 +405,18 @@ message StatSummary {
   repeated string statistics = 2;
 }
 
+// Computes basic statistics for numeric and string columns, including count, mean, stddev, min,
+// and max. If no columns are given, this function computes statistics for all numerical or
+// string columns.
+// It will invoke 'Dataset.describe' to compute the results.
```

Comment:
`It will invoke 'Dataset.describe' to compute the results.`

I tend to avoid say this as this exposes implementation details while the comment here actually serves as proto documentation. (E.g. we might switch the implementation to something else but forget to update here thus cause inconsistency)

Reply:
it has been removed.